Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -156,6 +159,9 @@
     private static final Logger logger = Logger.getLogger(Case.class.getName());
     private static final AutopsyEventPublisher eventPublisher = new AutopsyEventPublisher();
     private static final Object caseActionSerializationLock = new Object();
+    private static Future<?> backgroundOpenFileSystemsFuture = null;
+    private static final ExecutorService startIngestJobsExecutor
```

Comment:
Sure.  Would something like `backgroundOpenFileSystemsExecutor` work?  Is the thread name okay?

Reply:
That name is a little lengthy, I'd take "background" out of the names since that is pretty clearly implied.

The thread name looks fine. You might check to see if the convention has been to not capitalize thread names, I have a feeling this is so, but I could be wrong.

Reply:
Okay.  I believe this has been corrected now.